src/main/java/com/SAS/soccer_association_system/game/Game.java

```diff
@@ -0,0 +1,240 @@
+package com.SAS.soccer_association_system.game;
+
+import java.time.LocalDate;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Game {
+
+    private Season season;
+    private LocalDate date;
+    private Team host;
+    private Team guest;
+    private int hostScore;
+    private int guestScore;
+    private Stadium stadium;
+    private GameEventslogger events;
+    private GameReport gameReport;
+    private List<Referee> referees;
+
+    /**
+     * Empty constructor
+     */
+    public Game() {
+        referees = new LinkedList<Referee>();
+    }
+
+    /**
+     * The function returns the winning team name. else, return draw.
+     *
+     * @return
+     */
+    public String getWinningTeam() {
+        if (hostScore > guestScore) {
+            return hostScore.name;
+        }
+
+        if (hostScore < guestScore) {
+            return guest.name;
+        }
+
+        return "draw";
+    }
+
+    /**
+     * The function starts the match
+     */
+    public void startGame() {
+        System.out.println("Game Started");
+    }
+
+    //TODO: implement endGame method
+    public void endGame() {
+    }
+
+    //TODO: implement editReport
+    public void editReport() {
```

Comment:
Maybe just get eventToChange and newEvent and replace the eventToChange with newEvent?

Reply:
done